app/models/s_s_j.rb

```diff
@@ -0,0 +1,5 @@
+module SSJ
```

Comment:
jsut FYI you should be able to write the file as ssj.rb

Reply:
<img width="676" alt="image" src="https://user-images.githubusercontent.com/1683182/201704687-8deae83b-c160-41a3-a7d3-58c0ffed1abe.png">

Maybe `rails zeitwerk:check` is more fickle?